javascript/javascript1/week2/cloths.js

```diff
@@ -0,0 +1,11 @@
+var cloths = ['tshirt and short','jacket and pants'];
+function temperature(temp) {
+    if (temp < 15){
+        return cloths = 'jacket and pants';
+    }
+    else {
+        return cloths = 'tshirt and short';
+    }
+}
```

Comment:
The array `cloths` has two values.
* 'tshirt and short'
* 'jacket and pants'
You should return one of these values.
```suggestion
    if (temp < 15){
        return cloths[0];
    }
    else {
        return cloths[1];
    }
}
```